src/patterns/ec2-app/base.ts

```diff
@@ -377,6 +452,121 @@ export class GuEc2App extends Construct {
       }
     }
 
+    if (props.googleAuth?.enabled) {
+      const prefix = `/${scope.stage}/${scope.stack}/${app}`;
+
+      const {
+        allowedGroups = ["engineering@guardian.co.uk"],
+        credentialsSecretsManagerPath = `${prefix}/google-auth-credentials`,
+      } = props.googleAuth;
+
+      if (allowedGroups.length < 1) {
+        throw new Error("googleAuth.allowedGroups cannot be empty!");
+      }
+
+      if (allowedGroups.find((group) => !group.endsWith("@guardian.co.uk"))) {
+        throw new Error("googleAuth.allowedGroups must use the @guardian.co.uk domain.");
+      }
+
+      const deployToolsAccountId = StringParameter.fromStringParameterName(
+        scope,
+        "deploy-tools-account-id-parameter",
+        NAMED_SSM_PARAMETER_PATHS.DeployToolsAccountId.path
+      );
+
+      // See https://github.com/guardian/cognito-gatekeeper for the source code
```

Comment:
Looks like the repo has been renamed? Should we also consider making the repo public?

Reply:
Thanks - yep renamed for clarity. Will update this. I kept it private as it gives details of key security behaviour, but we can raise at Infosec catchup perhaps.